content/blog/2025/binder-singlenode/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "An experiment to support `mybinder.org` with a single node federation member"
+date: "2025-01-18"
+authors: ["Yuvi Panda", "Chris Holdgraf"]
+tags: [open source]
+categories: [impact]
+featured: false
+draft: false
+---
+
+{{% callout note %}}
+If you're interested in supporting `mybinder.org` with cloud resources, financial resources, or human resources, please see the [Support Binder](https://mybinder.readthedocs.io/en/latest/about/support.html) page for how you can help.
```

Comment:
I've just noticed the `Donate funds` box on https://mybinder.readthedocs.io/en/latest/about/support.html still links to Numfocus.

Reply:
updated that here, thanks for catching it:

- https://github.com/jupyterhub/mybinder.org-user-guide/pull/305